packages/cactus-core-api/src/main/typescript/plugin/ledger-connector/i-socket-api-client.ts

```diff
@@ -8,15 +8,19 @@ import type { Observable } from "rxjs";
  */
 export interface ISocketApiClient<BlockType> {
   sendAsyncRequest?(
-    contract: Record<string, unknown>,
-    method: Record<string, unknown>,
     args: any,
+    method?: Record<string, unknown>,
+    methodName?: string,
+    baseConfig?: any,
+    contract?: Record<string, unknown>,
   ): void;
 
   sendSyncRequest?(
-    contract: Record<string, unknown>,
-    method: Record<string, unknown>,
     args: any,
+    method?: Record<string, unknown>,
```

Comment:
@StepniowskiP Seems like a breaking change. Could you please refactor it to be backwards compatible with `1.0.0`?

Reply:
Sure, I have refactored method parameter to one before my changes